react/components/ExtensionPointFunctionComponent.tsx

```diff
@@ -61,16 +61,21 @@ const LazyLoadedExtensionPointComponent = (props: any) => {
   const isRootTreePath = treePath.indexOf('/') === -1
   const isAround = treePath.indexOf('$around') !== -1
 
-  const [Component, setComponent] = useState(
-    (component && getImplementation(component)) || null
-  )
+  const initialComponent = (component && getImplementation(component)) || null
+
+  const [Component, setComponent] = useState(initialComponent)
```

Comment:
It's possible to use a function in the initial state, so `getImplementation` isn't run more than once.
```suggest
const [Component, setComponent] = useState(() => (component && getImplementation(component)) || null)
```